charts/aws-efs-csi-driver/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 appVersion: "1.2.0"
 name: aws-efs-csi-driver
 description: A Helm chart for AWS EFS CSI Driver
-version: 1.2.2
-kubeVersion: ">=1.17.0-0"
+version: 1.3.0
```

Comment:
do we need to bump the major version? I'm not really sure about best practice but I think yes?

Reply:
In the apst, we have bumped minor version to signal that a new efs driver version is out.

So e.g. currently it so happens that 1.2.x correspondx to efs driver 1.2.y.

That's why I think we need a major version bump

Reply:
I am creating breaking changes in the values file, that's why I thought about bumping the major version. I would say most helm charts versions don't match the app version.

Reply:
Good idea since there is a lot of potential breaking changes! Let's bump to `2.0.0` to make it significant.